docbook/src/docbkx/autoscale-devguide.xml

```diff
@@ -57,9 +57,9 @@
         <releaseinfo>API v1.0</releaseinfo>
         <productname>&PRODNAME;</productname>
         <!-- enter the publication date of this document; in revhistory, explain what changed -->
-        <pubdate>2014-12-04</pubdate>
+        <pubdate>2015-01-14</pubdate>
         <copyright>
-            <year>2014</year>
+            <year>2015</year>
```

Comment:
We had an initiative in DevDoc on this issue last year. The copyright is drawn from a template, so, yes, that's the way they want it. This change I made does not actually print out (I know because I tried to make it 2014-2015 and it was not taken).